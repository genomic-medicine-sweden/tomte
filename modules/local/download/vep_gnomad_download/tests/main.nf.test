nextflow_process {

    name "Test Process VEP_GNOMAD_DOWNLOAD"
    script "../main.nf"
    process "VEP_GNOMAD_DOWNLOAD"
    tag "modules"
    tag "modules_local"
    tag "vep_gnomad_download"

    test("vep_gnomad_download -stub") {

        when {
            options "-stub"
            process {
                """
                input[0] = 'GRCh38'
                input[1] = 110
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
