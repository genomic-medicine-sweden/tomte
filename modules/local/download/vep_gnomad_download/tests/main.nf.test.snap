{
    "vep_gnomad_download -stub": {
        "content": [
            {
                "0": [
                    [
                        "/home/lucia.penaperez/github/gms/tomte/.nf-test/tests/51e3b9d76b35db65f736ad3a8e40ff40/work/03/7a1948072f961cc16954661617b6e9/gnomad_v4.0.vcf.gz",
                        "gnomad_v4.0.vcf.gz.tbi:md5,d41d8cd98f00b204e9800998ecf8427e"
                    ]
                ],
                "1": [
                    "versions.yml:md5,b53047a306822256b15bf6ddb998a393"
                ],
                "gnomad_vcf_tbi": [
                    [
                        "/home/lucia.penaperez/github/gms/tomte/.nf-test/tests/51e3b9d76b35db65f736ad3a8e40ff40/work/03/7a1948072f961cc16954661617b6e9/gnomad_v4.0.vcf.gz",
                        "gnomad_v4.0.vcf.gz.tbi:md5,d41d8cd98f00b204e9800998ecf8427e"
                    ]
                ],
                "versions": [
                    "versions.yml:md5,b53047a306822256b15bf6ddb998a393"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.5"
        },
        "timestamp": "2025-04-23T08:51:23.334743"
    }
}